### PR TITLE
refactoring code to reduce cyclomatic complexity

### PR DESCRIPTION
**Pull request by mrjbtc**

### CODE REVIEW

File: lib/public/AppFramework/OCSController.php [hunk 1]
```diff
@@ -105,12 +105,11 @@ public function buildResponse($response, $format = 'xml') {
...
 	private function buildOCSResponse($format, DataResponse $data) {
+		$version = 'V2Response';
 		if ($this->ocsVersion === 1) {
-			$response = new \OC\AppFramework\OCS\V1Response($data, $format);
-		} else {
-			$response = new \OC\AppFramework\OCS\V2Response($data, $format);
+			$version = 'V1Response';
 		}
-
+		$response = new \OC\AppFramework\OCS\$version($data, $format);
```
Comment (rullzer) [inline lib/public/AppFramework/OCSController.php]: This is not valid

**Review by rullzer — CHANGES_REQUESTED** [review]

**Comment by rullzer** [comment]
@noodlesegg thanks for you contribution!
You need to sign off your commits see: https://github.com/nextcloud/server/blob/master/contribute/developer-certificate-of-origin

File: lib/public/AppFramework/OCSController.php (hunk 1 above)
Comment (noodlesegg) [inline lib/public/AppFramework/OCSController.php]: oops will change it..

**Review by rullzer — APPROVED** [review]

**Comment by rullzer** [comment]
@noodlesegg could you squash your commits and sign off?

**Comment by noodlesegg** [comment]
hi @rullzer I'm sorry but how can I squash my commits and sign it off? thanks

**Comment by rullzer** [comment]
1. `git reset --soft HEAD~2`
2. `git commit -s'
3. `git push -f`

At step 2 you'll have to enter the commit message.

**Comment by MorrisJobke** [comment]
Beside the comment of @rullzer - 👍 from me as well :)

**Comment by rullzer** [comment]
Replaced by #2032

**Pull request closed without merging**